app/main.py

```diff
@@ -1,3 +1,49 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    km: int
```

Comment:
km is instance attribute. Remove it from here

Reply:
done